jax/_src/numpy/reductions.py

```diff
@@ -433,13 +433,19 @@ def _average(a: ArrayLike, axis: Axis = None, weights: ArrayLike | None = None,
 @implements(np.var, skip_params=['out'])
 def var(a: ArrayLike, axis: Axis = None, dtype: DTypeLike | None = None,
         out: None = None, ddof: int = 0, keepdims: bool = False, *,
-        where: ArrayLike | None = None) -> Array:
-  return _var(a, _ensure_optional_axes(axis), dtype, out, ddof, keepdims,
+        where: ArrayLike | None = None, correction: int | float = 0) -> Array:
+  if ddof != 0:
+    if correction != 0:
+      raise ValueError(
```

Comment:
@Micky774 we should check somewhere that this error is raised?
This code of checking ddof and correction usage is a bit biased though as both have 0 default value, so this code wont complain: `var(..., ddof=0, correct=0, ...)`

Reply:
For now we could have something like this
```python
class DefaultArg:
    def __init__(self, val=None, msg=None):
        self.val = val
        self.msg = msg

    def __repr__(self):
        return "<no value>" if self.msg is None else self.msg

def slope(rise, run=DefaultArg(1, "1"), delta_x=DefaultArg()):
    if isinstance(run, DefaultArg):
        run = run.val if isinstance(delta_x, DefaultArg) else delta_x
    elif not isinstance(delta_x, DefaultArg):
        raise ValueError("Only one of `run` or `delta_x` can be set at a time.")
    del delta_x

    return rise / run
```

In our case, we'd set `ddof=DefaultArg(0, "0"), correction=DefaultArg()`, which would mean that we can ignore `correction` unless it is specifically set, and as a bonus our docs would match how NumPy docs render this as well. What do you think?

With that being said, I really dislike the idea of having two parallel names for the same functionality...but I don't think we can afford to drop `ddof` since likely a large numpy userbase will still rely on that arg.

Reply:
Yes, this can be a way to workaround. I wonder whether this approach could be compatible with array API and better than another approach where we could simply set `correction: int | float | None = None` and check for None to raise the error ?

Reply:
This approach is still compatible with the array API. I only worry about folks that may quickly read "correction=None" in documentation and get confused, despite the parameter description being explicit.

Reply:
We can improve the docs if it only docs problem?
IMO, it would the same as for numpy where:
```
numpy.std(a, axis=None, dtype=None, out=None, ddof=0, keepdims=<no value>, *, where=<no value>, mean=<no value>, correction=<no value>)
...
correction{int, float}, optional
    Array API compatible name for the ddof parameter. Only one of them can be provided at the same time.
```